scss/_o-carousel.scss

```diff
@@ -47,10 +47,6 @@
   /* rtl:raw:
   transform: rotate(180deg);
   */
-
-  &::after {
```

Comment:
@isabellechanclou Without those 2 `&::after` rules we can see blue arrows:

![Screenshot from 2022-07-21 08-04-39](https://user-images.githubusercontent.com/17381666/180141023-19f52f96-5d60-4eb6-8beb-82415e5a1e85.png)

Reply:
Reintroduced CSS via 886dcbba351b0ca81787112d148e6f57ece9264a